src/Video.tsx

```diff
@@ -194,8 +194,8 @@ const Video = forwardRef<VideoRef, ReactVideoProps>(
         return;
       }
       const type = typeof selectedTextTrack.value;
-      if (type !== 'number' && type !== 'string') {
-        console.log('invalid type provided to selectedTextTrack');
+      if (type !== 'number' && type !== 'string' && type !== 'undefined') {
+        console.log('invalid type provided to selectedTextTrack.value');
```

Comment:
```suggestion
        console.log('invalid type provided to selectedTextTrack:', type);
```

Reply:
It actually is the type of the `selectedTextTrack.value` property that this refers to. But it was confusing because there is also a `.type` property but the `type` variable refers to the type of the `.value` propery. So to avoid confusion in future I have renamed the variable to `typeOfValueProp`.